### PR TITLE
Mixin simplification/V-model capability

### PR DESCRIPTION
**Pull request by marina-mosti**
I think we should keep the FormMixin unopinionated, having a required label property is fine for the provided example but if users are going to be able to pull this mixin into their own components then we shouldn't assume that every input is going to have a required label property. Maybe they want to do that as a slot, or with some other configuration.

I've moved the prop to the example components and removed it from the mixin, but open for comments.

### CODE REVIEW

**Comment by marina-mosti** [comment]
Added some changes to be able to add v-modeling to SchemaForm, also subject to opinion. In any case, if the user ends up doing things like on `SimpleExample.vue` everything will still work the same.

**Comment by marina-mosti** [comment]
Moved all the styles set on the FormSchema to an external stylesheet for the docs to leave that component without any implicit styles

**Comment by marina-mosti** [comment]
Added nested schema recursion. There's probably some refactoring to be done, its kind of hacky with all the if checks

File: src/SchemaWizard.vue [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    }
+  },
+  computed: {
+    currentSchema () {
+      return this.schema[this.step]
+    }
+  },
+  methods: {
+    update (data) {
+      this.$set(this.wizardValues, this.step, data)
```
Comment (shentao) [inline src/SchemaWizard.vue]: I think we can get rid of the `this.wizardValues` with a `map`.
What do you think of the below?
```
const wizardValues = this.value.map((stepValue, index) => {
  return index === this.step
    ? data
    : stepValue
})

this.$emit('input', wizardValues)
```

File: docs/.vuepress/components/ExampleVModel.vue [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  },
+  lastName: {
+    component: 'FormText',
+    label: 'Last Name',
+  },
+  email: {
+    component: 'FormText',
+    label: 'Your email',
+    required: true,
+    config: {
```
Comment (NataliaTepluhina) [inline docs/.vuepress/components/ExampleVModel.vue]: Do we have any established opinion on what should belong to `config` and what shouldn't?

Comment (shentao) [reply docs/.vuepress/components/ExampleVModel.vue]: The way I see it, `config` is a catch-all property, that is being "v-bound" to the component as it is, thus allowing for example to pass custom props to custom components.

File: src/SchemaWizard.vue (hunk 1 above)
Comment (marina-mosti) [reply src/SchemaWizard.vue]: Done, found a (in my opinion) simpler solution

**Pull request merged**